addons/l10n_qa/i18n/qa_QA.po

```diff
@@ -0,0 +1,21 @@
+# Translation of Odoo Server.
```

Comment:
I'm not sure of the name of this file. According to [this](https://www.localeplanet.com/icu/iso639.html#ar), `ar_001` is standard Arabic which I think is the one we should use while the one for Qatar is `ar_QA`. What is `qa_QA` supposed to be?